TappNetwork/home-visiting-app-web
# Cu 2pthyprx nav bar

# Description
Replaced the nav bar on template code on every page with a new nav bar component that displays a menu toggle button when menu is hidden.

# Steps to test changes

1. visit /login
2. Resize page so that navbar disappears
3. Confirm that menu toggle button appears on nav bar and click it to toggle menu. 
4. Navigate to every page to ensure bar is rendering properly on each.

# Issues Addressed

[2pthypr](https://app.clickup.com/t/2pthypr)

# Screenshots
<img width="720" alt="Screen Shot 2022-08-12 at 7 29 35 AM" src="https://user-images.githubusercontent.com/29612767/184363933-aa0d1406-257b-46d6-a41d-cb0a52f15594.png">
